### ISSUE TITLE
Extend example to utilize Spring IoC container

### ISSUE THREAD

**Issue by hstaudacher** [bwolff/rwt-on-spring-boot#1]
I really like the example. Bring RWT and Spring Boot together is a great showcase. While browsing the code I wondered how would you suggest to make use of the [Spring IoC container](https://www.baeldung.com/inversion-control-and-dependency-injection-in-spring) aka. inject dependencies.

It seems that with the entry point structure you are building an object graph in parallel to the one managed by Spring Boot. It would be very nice to have an example that show e.g. how to inject a "UI component" that makes use of some other beans. 

Just in case you can spare some time :).

Cheers Holger

**Comment by bwolff** [comment]
Hi Holger,

glad to hear that you like the example.

Yes, it would be nice to have an example how to complete wire together the application using more Spring IoC. I will extend the example with this when I find the time.

Cheers
Ben
